fnirs_tapping/dataset_description.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Name": "fNIRS finger tapping experiment",
+    "BIDSVersion": "1.6.0",
```

Comment:
Assuming that NIRS will be merged into the specification next, this would be merged after the 1.8.0 release

```suggestion
    "BIDSVersion": "1.8.0",
```